### ISSUE TITLE
New bot function

### ISSUE THREAD

**Issue by GreYzZ** [TheAnthonyNL/SteamBot#22]
So if the 10 is at 10 Seconds not users can go in anymore.
It's good that that people can't snipe so easy.

**Comment by Xeroner** [comment]
??

**Comment by GreYzZ** [comment]
@Xeroner 
If the timer comes to the last 10 Seconds bot won't accept any more trades

**Comment by Xeroner** [comment]
Becouse of steaminv delay i think.

**Comment by GreYzZ** [comment]
@Xeroner 
Yes and because that the bot can roll at Timer 0 and dont have to accept more trades

**Comment by Xeroner** [comment]
@GreYzZ There should me 10 second for accept last tades or something..
